### DISCUSSION TITLE
v0.19.3 release

### DISCUSSION THREAD

**Discussion by Conengmo** [python-visualization/folium#2062] [Collaborators]
@hansthen any comments on the pending release v0.19.3? https://github.com/python-visualization/folium/releases/tag/untagged-191931d9cd2f877252f9 I hope this helps users avoid getting issues when type checking their code that uses Folium.

**Comment by hansthen** [comment]
I think the release notes are fine. Let's do it!

**Reply by Conengmo** [reply]
Thanks!
